tests/unit/test_events.py

```diff
@@ -83,6 +84,12 @@ def test_formatting(self):
         event = types.JinjaLogDebug(msg=[1, 2, 3])
         assert isinstance(event.msg, str)
 
+    def test_set_adapter_dependency_log_level(self):
```

Comment:
There's no assertion in this test. Are we just making sure that this runs without error?